bioio_base/standard_metadata.py

```diff
@@ -0,0 +1,109 @@
+from dataclasses import dataclass
+from typing import Optional, Sequence
+
+
+@dataclass
+class StandardMetadata:
+    """
+    A simple container for embedded metadata fields using dataclass.
+
+    Each metadata field is defined with an optional type.
+    The FIELD_LABELS mapping is used to produce readable output via the to_dict method.
+    """
+
+    # Acquisition date.
+    acquisition_date: Optional[str] = None
+
+    # Binning configuration.
+    binning: Optional[str] = None
+
+    # Column information.
+    column: Optional[str] = None
+
+    # List or sequence of dimension names.
+    dimensions_present: Optional[Sequence[str]] = None
+
+    # Channel dimension size.
+    image_size_c: Optional[int] = None
+
+    # Time dimension size.
+    image_size_t: Optional[int] = None
+
+    # Spatial X dimension size.
+    image_size_x: Optional[int] = None
+
+    # Spatial Y dimension size.
+    image_size_y: Optional[int] = None
+
+    # Spatial Z dimension size.
+    image_size_z: Optional[int] = None
+
+    # The experimentalist who produced this data.
+    imaged_by: Optional[str] = None
+
+    # Date this file was imaged.
+    imaging_date: Optional[str] = None
+
+    # Objective.
+    objective: Optional[str] = None
+
+    # Physical pixel size along X.
+    pixel_size_x: Optional[float] = None
+
+    # Physical pixel size along Y.
+    pixel_size_y: Optional[float] = None
+
+    # Physical pixel size along Z.
+    pixel_size_z: Optional[float] = None
+
+    # Position index, if applicable.
+    position_index: Optional[int] = None
+
+    # Row information.
+    row: Optional[str] = None
+
+    # Is the data a timelapse?
+    timelapse: Optional[bool] = None
+
+    # Time interval between frames.
+    timelapse_interval: Optional[float] = None
+
+    # Total time duration of imaging.
+    total_time_duration: Optional[str] = None
+
+    # Mapping of internal attribute names to readable labels.
+    FIELD_LABELS = {
+        "acquisition_date": "Acquisition Date",
+        "binning": "Binning",
+        "column": "Column",
+        "dimensions_present": "Dimensions Present",
+        "image_size_c": "Image Size C",
+        "image_size_t": "Image Size T",
+        "image_size_x": "Image Size X",
+        "image_size_y": "Image Size Y",
+        "image_size_z": "Image Size Z",
+        "imaged_by": "Imaged By",
+        "imaging_date": "Imaging Date",
```

Comment:
what's the difference between acquisition date and imaging date?

Reply:
I think Imaging Date is the time ignorant date stamp of when the image was taken (started?) and the Acquisition Date is per timepoint. Is that what you were going for here Brian?

Reply:
acquisition date is what MXS extracts and imaging date is what was in labkey. Theyre the same from what I can tell. I removed acquisition date in [d739331](https://github.com/bioio-devs/bioio-base/pull/29/commits/d739331e5756e035e795113a19359223f54f5b81)